aikau/src/main/resources/alfresco/forms/controls/NumberSpinner.js

```diff
@@ -161,30 +224,59 @@ define(["alfresco/forms/controls/BaseFormControl",
        */
       isNumberValidator: function alfresco_forms_controls_FormControlValidationMixin__isNumberValidator(validationConfig) {
          var isValid = false;
-         try {
-            var value = this._removeCommasAndSpaces(this.wrappedWidget.textbox.value),
-               isNumber = this._valueIsNumber(value);
-            isValid = isNumber || (this.permitEmpty && value === "");
-         } catch (e) {
+         try 
+         {
+            var textBoxValue = this.wrappedWidget.textbox.value;
```

Comment:
I disagree here... for the simple reason that if you always return null, then it's not possible to tell whether or not the value is NaN or if it is just blank - which means that it's not possible to perform the permitEmpty test.

Let's also try to not lose sight of the fact that we **need** this to work, and it **does** work.